yellowbrick/cluster/silhouette.py

```diff
@@ -18,6 +18,7 @@
 ##########################################################################
 
 import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
See my comment below about using `import matplotlib.ticker as ticker` rather than `import matplotlib.pyplot as plt` and let me know what you think.